src/main/java/com/comphenix/protocol/wrappers/WrappedLevelChunkData.java

```diff
@@ -0,0 +1,591 @@
+                    && Objects.equals(emptyBlockYMask, lightData.emptyBlockYMask)
+                    && Arrays.deepEquals(skyUpdates.toArray(), lightData.skyUpdates.toArray())
+                    && Arrays.deepEquals(blockUpdates.toArray(), lightData.blockUpdates.toArray());
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(skyYMask, blockYMask, emptySkyYMask, emptyBlockYMask, skyUpdates, blockUpdates, trustEdges);
+        }
+
+        @Override
+        public String toString() {
+            return "LightData{" +
+                    "skyYMask=" + skyYMask +
+                    ", blockYMask=" + blockYMask +
+                    ", emptySkyYMask=" + emptySkyYMask +
+                    ", emptyBlockYMask=" + emptyBlockYMask +
+                    ", skyUpdates=" + skyUpdates +
+                    ", blockUpdates=" + blockUpdates +
+                    ", trustEdges=" + trustEdges +
+                    '}';
+        }
+
+        private static ConstructorAccessor lightUpdatePacketDataConstructor;
+
+        private static List<FieldAccessor> bitSetAccessors;
+        private static List<FieldAccessor> byteArrayListAccessors;
+
+        private static FieldAccessor trustEdgesAccessor;
+
+        public static EquivalentConverter<LightData> getConverter() {
+            return new EquivalentConverter<LightData>() {
+
+                @Override
+                public Object getGeneric(LightData specific) {
+                    if(lightUpdatePacketDataConstructor == null) {
+                        lightUpdatePacketDataConstructor = Accessors.getConstructorAccessor(HANDLE_TYPE, MinecraftReflection.getPacketDataSerializerClass(), int.class, int.class);
+                    }
+
+                    ByteBuf byteBuf = Unpooled.buffer();
+
+                    byteBuf.writeBoolean(specific.trustEdges);
+
+                    serializeBitSet(byteBuf, specific.skyYMask);
+                    serializeBitSet(byteBuf, specific.blockYMask);
+                    serializeBitSet(byteBuf, specific.emptySkyYMask);
+                    serializeBitSet(byteBuf, specific.emptyBlockYMask);
+                    serializeByteArrayList(byteBuf, specific.skyUpdates);
+                    serializeByteArrayList(byteBuf, specific.blockUpdates);
+
+                    return lightUpdatePacketDataConstructor.invoke(MinecraftReflection.getPacketDataSerializer(byteBuf), 0, 0);
+                }
+
+                private void serializeByteArrayList(ByteBuf byteBuf, List<byte[]> bytes) {
+                    WirePacket.writeVarInt(byteBuf, bytes.size());
+
+                    for (byte[] entry : bytes) {
+                        WirePacket.writeVarInt(byteBuf, entry.length);
+
+                        byteBuf.writeBytes(entry);
+                    }
+                }
+
+                private void serializeBitSet(ByteBuf byteBuf, BitSet bitSet) {
+                    long[] toSerialize = bitSet.toLongArray();
+
```

Comment:
I'd say remove whitespace here too as you are using the variable one line below anyway